opts/secret.go

```diff
@@ -53,10 +52,6 @@ func (o *SecretOpt) Set(value string) error {
 		case "source", "src":
 			options.SecretName = value
 		case "target":
-			tDir, _ := filepath.Split(value)
```

Comment:
The client code has no way of knowing whether a path is a file or a directory.

This old code was essentially checking that there was no path separator. It is being removed because now path separators are allowed.